### DISCUSSION TITLE
Why are there no description options or helper text for tabs in forms?

### DISCUSSION THREAD

**Discussion by jcc5018** [filamentphp/filament#12436] [Help]
### Package

Form builder

### Package Version

v3.2

### How can we help you?

why are certain helper items like descriptions not available on every layout item or component? I'd edit it myself but your package is rather complicated and i dont fully understand how to do so.  Can someone help me out, and also let me know how i go about adding paragraph breaks 

```
   Tabs\Tab::make('Sample tab')
                                ->schema([
                                    Toggle::make('field1')
                                          ->helperText('text here'),
                                    Toggle::make('field 2'),
                                ])->description('tab helper text'),
```

**Comment by Fanom2813** [comment]
i wish we had that ability
